src/net.cpp

```diff
@@ -2695,6 +2710,33 @@ bool CConnman::Start(CScheduler& scheduler, const Options& connOptions)
 {
     Init(connOptions);
 
+    if (strSocketEventsMode.empty()) {
+#if defined USE_POLL
+        strSocketEventsMode = "poll";
+#else
+        strSocketEventsMode = "select";
+#endif
+    } else {
+        std::set<std::string> supported;
+#ifdef USE_POLL
+        supported.emplace("poll");
+#endif
+        supported.emplace("select");
+        if (!supported.count(strSocketEventsMode)) {
+            std::string strSupported;
+            for (auto& s : supported) {
+                if (!strSupported.empty()) {
+                    strSupported += ", ";
+                }
+                strSupported += s;
+            }
+            clientInterface->ThreadSafeMessageBox(
```

Comment:
should wrap in `if (clientInterface) {`

Reply:
Implicitly fixed by removing this code (this logic is in init.cpp now)